CMake/Packages.cmake

```diff
@@ -2,10 +2,10 @@
 if(APPLE)
   add_custom_target(
     packages
-    "${CMAKE_SOURCE_DIR}/tools/deployment/make_osx_package.sh"
+    COMMAND "${CMAKE_SOURCE_DIR}/tools/deployment/make_osx_package.sh" -k
     WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
     COMMENT "Building default OS X package (no custom config)" VERBATIM
-    DEPENDS daemon shell
+    DEPENDS daemon shell kernel-build
```

Comment:
Is it possible to add the `-k` and `kernel-build` iff a kernel has been built? Maybe we keep the `-k` but it will silently succeed if there's no kernel output. The reason I ask is because we don't want to interrupt folks who regularly build their own packages but will not want to build the kernel components. And I just confirmed that the folks who are building their own packages use `make packages`.